app/controllers/tool_groups_controller.rb

```diff
@@ -2,7 +2,13 @@ class ToolGroupsController < ApplicationController
   before_action :authorize!
 
   def index
-    render json: tool_groups_ordered_by_name, status: :ok
+    include = {}
+    fields = {}
+
+    include = params[:include]&.split(",") if params[:include]
```

Comment:
I'm surprised you need to manually split on the `,`, we haven't had to do that with our other controllers and regularly request multiple included relationships from those endpoints

Reply:
You are rigth, running tests without this split there works fine